void_migration/plotter.py

```diff
@@ -103,20 +103,31 @@ def plot_s_bar(y, s_bar, nu_time, p):
 def plot_s(x, y, s, p, t):
     plt.figure(fig)
     plt.clf()
+    ##################### if charge_discharge and mono-disperse #####################################
+    colors = [(1, 0, 0), (0, 0, 1)]
+    cmap_name = 'my_list'
+    cmap = LinearSegmentedColormap.from_list(cmap_name, colors, N=2)
+    #################################################################################################
     with warnings.catch_warnings():
         warnings.simplefilter("ignore", category=RuntimeWarning)
         s_plot = np.nanmean(s, axis=2).T
     s_plot = np.ma.masked_where(np.isnan(s_plot), s_plot)
-    plt.pcolormesh(x, y, s_plot, cmap=orange_blue_cmap, vmin=0.001, vmax=0.01)
+    # plt.pcolormesh(x, y, s_plot, cmap=orange_blue_cmap, vmin=p.s_m, vmax=p.s_M)
```

Comment:
this change will break all other use cases of this function. can you either make it a new command, or even better move it into the existing `plot_c` function, and have an optional function argument to choose between using the `c` variable or a fixed number of `s` increments?